mindsdb/integrations/handlers/notion_handler/README.md

```diff
@@ -0,0 +1,94 @@
+# Notion Handler
+
+This is the implementation of the Notion handler for MindsDB.
+
+## Notion
+Notion is a note-taking and productivity freemium cloud platform.
+In short, notion has all-in-one workspace tool that integrates kanban boards, tasks, wikis, and database.
+In this handler. python client of api is used and more information about this python client can be found [here](https://pypi.org/project/notion-client/)
+
+
+## Implementation
+This handler was implemented as per the MindsDB API Handler documentation.
+
+
+The required arguments to establish a connection are,
+* `notion_api_token`: API key for acessing the Notion instance.
+
+
+## Usage
+In order to make use of this handler and connect to an Notion in MindsDB, the following syntax can be used,
+
+```sql
+CREATE DATABASE notion_source
+WITH
+engine='notion',
+parameters={
+    "api_token": "<notion-api-token>",
+};
+```
+
+## Implemented Features
+
+Now, you can use this established connection to query your table as follows,
+
+### Database
```

Comment:
If it is mandatory to provide the IDs, can we mention that in the README as well, please?